magefiles/lint.go

```diff
@@ -37,9 +38,13 @@ func (Lint) Yaml() error {
 	if err != nil {
 		return err
 	}
-	return sh.RunV("docker", "run", "--rm",
+	// Run prettier for yaml formatting
+	prettierErr := sh.RunV("docker", "run", "--rm", "-w", "/src", "-v", fmt.Sprintf("%s:/src:ro", cwd), "ghcr.io/tmknom/dockerfiles/prettier:3.6.2", "-c", ".")
+	// Run yamllint for content checking
+	yamllintErr := sh.RunV("docker", "run", "--rm",
 		"-v", fmt.Sprintf("%s:/src:ro", cwd),
 		"cytopia/yamllint:1", "-c", "/src/.yamllint", "/src")
+	return errors.Join(prettierErr, yamllintErr)
```

Comment:
Running both of these linters in a single command.

Reply:
i'm confused, your PR description made it sound like you were getting rid of `yamllint` altogether..